src/dms/core/EdFi.DataManagementService.Core/ApiSchema/ApiSchema_Schema.json

```diff
@@ -52,6 +52,7 @@
                     },
                     "additionalProperties": false
                 },
+                "compatibleDsRange": true,
```

Comment:
This is a new value in apischema for extensions. It comes in as `null` currently. Adding this line prevents schema validation errors.